docs/_series/lilphockers.md

```diff
@@ -3,11 +3,87 @@ name: LILPHOCKERS
 author: Whoisit
 image: https://gf5ueyjsow7vyox4gglcuzsqdcmwxspphgyh6oplsyid7ogvbsma.arweave.net/MXtCYTJ1v1w6_DGWKmZQGJlrye85sH8565YQP7jVDJg/LILPHOCKERSINTRO.gif
 date: 2022-02-17
-description:
+description: Owners of the grail card will own all current and upcoming subs. Subs will be sent out once the supply of the grail card is fully distributed or the remainder is burnt.
 subs: 
+  -
+    name: BUBBLES
+    image: https://arweave.net/zy8CF_XUiFBmRJVXdJs0qhVvbqc4X8NUymOeApZtLyk/nzck2t_image.gif
+    supply: 100
+  -
+    name: RICK
+    image: https://qsoayg3racqaeahgdu7mlqbslhy2op7stzlrehda7zkttfzemnda.arweave.net/hJwMG3EAoAIA5h0-xcAyWfGnP_KeVxIcYP5VOZckY0Y/Rick.gif
+    supply: 100
   -
     name: FINN
-    image: https://7bkm27psb4i6l3hk3eydefr57c3nofrpdcmk5deudjjxqd3mrzya.arweave.net/-FTNffIPEeXs6tkwMhY9-LbXFi8YmK6MlBpTeA9sjnA/LILPHOCKERS.FINN.gif
+    image: https://uvvqspkpkoyhcobwyaqkf3pbr5c27ojxn4fx2gopylc3amsi5e5a.arweave.net/pWsJPU9TsHE4NsAgou3hj0WvuTdvC30Zz8LFsDJI6To/Finn%20the%20Human.gif
+    supply: 100
+  -
+    name: JBRAVO
+    image: https://k27tdvkmo6uu5rodrwraicpophssln7qggzryxplf7vtrmdu5wca.arweave.net/Vr8x1Ux3qU7Fw42iBAnueeUlt_Axsxxd6y_rOLB07YQ/JBravo.gif
+    supply: 100
+  -
+    name: ANGELICA
+    image:https://arweave.net/7VEcZTWgLHWBSAycdvj5cWjoyJgmqNNJYoMtMRDyw_4/swf9f4_image.gif
```

Comment:
Key `image:` needs to be separated from the value (the link to the image) with space like so: 

`image: https://arw....`

Reply:
just discovered this section. handy. updated it